astropy/cosmology/io/latex.py

```diff
@@ -57,7 +57,7 @@ def write_latex(
     table = to_table(cosmology, cls=cls, cosmology_in_meta=False)
 
     cosmo_cls = type(cosmology)
-    for name, col in table.columns.copy().items():
+    for name in tuple(table.colnames):
         param = getattr(cosmo_cls, name, None)
         if not isinstance(param, Parameter) or param.unit in (None, u.one):
             continue
```

Comment:
It is possible to get rid of the `continue` here by inverting the condition in the if-statement.

Reply:
Yes. as 

```
cosmo_cls = type(cosmology)
    for name, col in table.columns.copy().items():
        param = getattr(cosmo_cls, name, None)
        if isinstance(param, Parameter) and param.unit is not None:
            table[name] <<= param.unit
```